### DISCUSSION TITLE
Error message about encryption although it is not enabled

### DISCUSSION THREAD

**Discussion by Feri-6** [Extended-Thunder/send-later#608] [General]
Hi,

I got the following error message with the latest (10.1) version:

"Send Later can't schedule messages that are encrypted or digitally signed; you must disable these on a draft before you can schedule it. You must also disable “Store draft messages in encrypted format” under “End-To-End Encryption” for accounts or identities for which you wish to schedule messages."

I use TB v102.14.0 and encryption, digital sign is not enabled for this account. Previous version of Send later worked well.

What is the solution?

Best regards,
Ferenc

**Comment by Bluestone357** [comment]
Me too - TB 115 + SL 10.0.9

**Reply by vmtitov00** [reply]
Cured. In "End-to-end encryption" of the account:

1. Create and apply a PGP key
2.  Uncheck "Store draft messages in encrypted format"

There is however a small problem that the extension saves the drafts but never sends them

**Reply by Bluestone357** [reply]
Sorted.
Key generated, applied, published.
SL also sends when expected.

**Reply by Onurb31** [reply]
Same error here.
I use TB 102.14.0 (32b).
My accounts are not encrypted. I have generated a OpenPGP key. I have unchecked "save drafts in encrypted formats". I have tried to send emails either by using the OpenPGP identity or not (but not signing or encrypting the messages). The message is saved n the draft folder with an hour corresponding to the hour on which it is to be sent. 
The SL box shows the message. However, it is never sent. I remains in the draft folder.
It is a pity that this error appears with the new update, as everything worked perfectly before this update.

**Comment by Piper526** [comment]
Same issue here but I don't use any encryption or digital signing. TB 102.14.0 (32-bit)

**Reply by jyunte** [reply]
Same here. Was working before the update, but not anymore. I have no encryption enabled.

**Reply by jyunte** [reply]
I tried adding a key, and even though it now adds the email to the drafts folder, it doesn't send the email at all.

**Comment by championshipworth** [comment]
Same issue here  TB 102.14.0  SL 10.1

**Comment by rosset37** [comment]
I'm also having the same problem. When I go into the Account Settings for Thunderbird, the options for disable encryption are already selected and greyed out. There is nothing for me to change or update like the error message instructs me to

**Comment by Spokey42** [comment]
same thing here.  Update is today 8/29.  Was working earlier today just stopped around 12:30 ET

No encryption, TB 102.14.0 

- - - edit - - - 
I can click the send now button that displays the dialog and send immediately.

**Comment by championshipworth** [comment]
Here is what I did and it worked.  If you have more than one email address you only need to do one email, the others will still work with SendLater

Click on your email address on the left pane.  To the right you will see end to end encryption.  Click on that.  Add a key to the email.  Scroll down and now the "Disable encryption for new message" can be selected.  

Scroll down to "Advanced Settings" and deselect under that heading. (this is what I did)

Then download this version to a file: [](https://addons.thunderbird.net/en-US/thunderbird/addon/send-later-3/versions/10.0.2)
Then go into the addon's page in thunderbird's addons screen, and click the cog in the top right. Click "Install addon from file" and select the file for 10.0.2 you just downloaded.

Have no idea why this worked, but it did.  Good luck and I hope we all figure this out.

**Comment by jyunte** [comment]
A bug fix is coming later today, per the developer, which will fix both issues - the encryption warning error, and the inability to send the email.

**Comment by jikamens** [comment]
Yeah, sorry about the trouble, there will be a fix later today.

**Comment by jikamens** [comment]
I have released version 10.1.1 of Send Later which fixes all of the issues people have encountered today. The full list of fixes in this version is viewable on its [GitHub release page][rp].

The new release has been uploaded to addons.thunderbird.net and is waiting for the moderators there to approve it. If you would rather not wait for them, you can get it as follows:

* Download [this file][xpi] and save it to your local computer. If you're reading this in Firefox, make sure to right-click on the link and select "Save Link As..." rather than just clicking on it, or Firefox will try to install it as a Firefox add-on.
* Open the Add-ons page in Thunderbird (Tools > Add-ons and Themes).
* Click the gear icon at the top.
* Select "Install Add-on from File...".
* Browse to the file you saved and select it.
* Click through the dialogs to confirm installation.
* You can delete the downloaded file once you've installed it.

If you don't want to do this you can wait until the moderators approve it, though I can't say whether that will take hours or days.

I'm sorry about these bugs and I thank you for your patience as I've fixed them as rapidly as I could.

[rp]: https://github.com/Extended-Thunder/send-later/releases/tag/v10.1.1
[xpi]: https://github.com/Extended-Thunder/send-later/releases/download/v10.1.1/send_later.xpi

**Reply by Blop3474** [reply]
Good morning,
Great to get this new version.
Many thanks for your prompt answer and release. 

Best regards

Lionel

**Reply by S-Zicari** [reply]
Thank you very much!

**Reply by Spokey42** [reply]
thx for the quick response/update/fix

**Comment by HDInfautre** [comment]
Thanks

**Comment by EvilRunner** [comment]
Thank you !!
